cc/util/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 tink_module(util)
 
+include(CheckSymbolExists)
```

Comment:
How about having these in tink/CMakeLists.txt?

Reply:
Done.